lib/daemon_runner/retry_errors.rb

```diff
@@ -0,0 +1,18 @@
+require 'retryable'
+
+module DaemonRunner
+    #
+    # Retry Errors
+    #
+    class RetryErrors
+      class << self
+        def retry(retries: 3, exceptions: [Faraday::ClientError], &block)
+          Retryable.retryable({
+            on: exceptions,
+            sleep: lambda { |c| Kernel.sleep(2 ** retries * 0.3) }},
```

Comment:
Thinking about it more you have a few options for how you want to handle this:
1. Pass the calculated retry directly to the `sleep` param.
2. Pass a lambda to the `sleep` param that does backoffs based on the number of retries.
3. Pass the lambda you wrote to the `sleep_method` param.